pkcs5/Cargo.toml

```diff
@@ -25,7 +25,7 @@ aes = { version = "=0.9.0-pre.2", optional = true, default-features = false }
 aes-gcm = { version = "=0.11.0-pre.2", optional = true, default-features = false, features = ["aes"] }
 des = { version = "=0.9.0-pre.2", optional = true, default-features = false }
 pbkdf2 = { version = "=0.13.0-pre.1", optional = true, default-features = false, features = ["hmac"] }
-rand_core = { version = "0.9.0", optional = true, default-features = false }
+rand_core = { version = "0.9.3", optional = true, default-features = false }
```

Comment:
These changes seem superfluous, should probably just change them to `0.9` so dependabot isn't constantly bumping them

Reply:
yeah good point.